include/natalie/object_type.hpp

```diff
@@ -4,11 +4,13 @@ namespace Natalie {
 
 enum class ObjectType {
     Nil,
+    EnumeratorArithmeticSequence,
```

Comment:
Can this be in alphabetical order with the rest? (Nil is at the top so it has a zero value.)